File: problem/ai-battle-series/16-ta-evaluation/src/main.py
import json
import sys


def main():
    payload = {
        "evaluations": [
            {
                "ta_name": "ChatGPT",
                "score": 8,
                "comment": "Clear and helpful explanations.",
                "strengths": ["Fast", "Friendly"],
                "improvements": ["Be more concise"],
            },
            {
                "ta_name": "Gemini",
                "score": 7,
                "comment": "Useful but sometimes inconsistent.",
                "strengths": ["Creative"],
                "improvements": ["Stay consistent"],
            },
            {
                "ta_name": "Opus",
                "score": 9,
                "comment": "Deep and structured guidance.",
                "strengths": ["Thorough"],
                "improvements": ["Shorten responses"],
            },
        ],
        "overall_comment":
        "Great support overall.",
        "would_recommend":
        True,
    }

    with open("evaluation.json", "w", encoding="utf-8") as f:
        json.dump(payload, f, ensure_ascii=False, indent=2)

    sys.stdout.write("Evaluation submitted successfully!\n")


if __name__ == "__main__":
    main()

File: problem/ai-battle-series/16-ta-evaluation/src/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ofstream fout("evaluation.json");
    if (fout) {
        fout << "{\n";
        fout << "  \"evaluations\": [\n";
        fout << "    {\"ta_name\": \"ChatGPT\", \"score\": 8, \"comment\": \"Clear and helpful explanations.\", \"strengths\": [\"Fast\", \"Friendly\"], \"improvements\": [\"Be more concise\"]},\n";
        fout << "    {\"ta_name\": \"Gemini\", \"score\": 7, \"comment\": \"Useful but sometimes inconsistent.\", \"strengths\": [\"Creative\"], \"improvements\": [\"Stay consistent\"]},\n";
        fout << "    {\"ta_name\": \"Opus\", \"score\": 9, \"comment\": \"Deep and structured guidance.\", \"strengths\": [\"Thorough\"], \"improvements\": [\"Shorten responses\"]}\n";
        fout << "  ],\n";
        fout << "  \"overall_comment\": \"Great support overall.\",\n";
        fout << "  \"would_recommend\": true\n";
        fout << "}\n";
    }
    cout << "Evaluation submitted successfully!\n";
    return 0;
}
